src/styles.scss

```diff
@@ -1466,3 +1466,8 @@ hr {
   height: 50px;
   object-fit: scale-down;
 }
+
+// material sometimes sets the font-weight to be bold (radio buttons for example) when we want it to be normal.
```

Comment:
if we want it to be normal, then it can be applied globally to all radio button groups or w/e instead of manually adding the class to each material component.